
[RUN MODE (2 stages)] (par-->seq)

***********************************<Summary of test-failures>***************************************
-------------------------------------------------------------------------------
Test set: org.apache.dubbo.config.spring.context.annotation.EnableDubboTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.994 s <<< FAILURE! - in org.apache.dubbo.config.spring.context.annotation.EnableDubboTest
testProvider  Time elapsed: 0.67 s  <<< ERROR!
org.apache.dubbo.rpc.RpcException: Fail to start server(url: dubbo://10.1.72.7:12345/org.apache.dubbo.config.spring.api.HelloService?anyhost=true&application=dubbo-demo-application&bind.ip=10.1.72.7&bind.port=12345&channel.readonly.sent=true&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&heartbeat=60000&interface=org.apache.dubbo.config.spring.api.HelloService&methods=sayHello&pid=69239&release=&side=provider&timestamp=1699889754350) Failed to bind NettyServer on /10.1.72.7:12345, cause: Address already in use
	at org.apache.dubbo.config.spring.context.annotation.EnableDubboTest.testProvider(EnableDubboTest.java:72)
Caused by: org.apache.dubbo.remoting.RemotingException: Failed to bind NettyServer on /10.1.72.7:12345, cause: Address already in use
	at org.apache.dubbo.config.spring.context.annotation.EnableDubboTest.testProvider(EnableDubboTest.java:72)
Caused by: java.net.BindException: Address already in use

testConsumer  Time elapsed: 0.322 s  <<< ERROR!
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'testConsumerConfiguration': Injection of @DubboReference dependencies is failed; nested exception is org.apache.dubbo.rpc.RpcException: Fail to start server(url: dubbo://10.1.72.7:12345/org.apache.dubbo.config.spring.api.DemoService?anyhost=true&application=dubbo-demo-application&bind.ip=10.1.72.7&bind.port=12345&channel.readonly.sent=true&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&heartbeat=60000&interface=org.apache.dubbo.config.spring.api.DemoService&methods=sayName,getBox&pid=69239&release=&revision=2.5.7&sayName.return=true&sayName.timeout=100&side=provider&timestamp=1699889754712&version=2.5.7) Failed to bind NettyServer on /10.1.72.7:12345, cause: Address already in use
	at org.apache.dubbo.config.spring.context.annotation.EnableDubboTest.testConsumer(EnableDubboTest.java:95)
Caused by: org.apache.dubbo.rpc.RpcException: Fail to start server(url: dubbo://10.1.72.7:12345/org.apache.dubbo.config.spring.api.DemoService?anyhost=true&application=dubbo-demo-application&bind.ip=10.1.72.7&bind.port=12345&channel.readonly.sent=true&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&heartbeat=60000&interface=org.apache.dubbo.config.spring.api.DemoService&methods=sayName,getBox&pid=69239&release=&revision=2.5.7&sayName.return=true&sayName.timeout=100&side=provider&timestamp=1699889754712&version=2.5.7) Failed to bind NettyServer on /10.1.72.7:12345, cause: Address already in use
	at org.apache.dubbo.config.spring.context.annotation.EnableDubboTest.testConsumer(EnableDubboTest.java:95)
Caused by: org.apache.dubbo.remoting.RemotingException: Failed to bind NettyServer on /10.1.72.7:12345, cause: Address already in use
	at org.apache.dubbo.config.spring.context.annotation.EnableDubboTest.testConsumer(EnableDubboTest.java:95)
Caused by: java.net.BindException: Address already in use

-------------------------------------------------------------------------------
Test set: org.apache.dubbo.config.spring.schema.GenericServiceTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.37 s <<< FAILURE! - in org.apache.dubbo.config.spring.schema.GenericServiceTest
testBeanDefinitionParser  Time elapsed: 0 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: java.lang.IllegalStateException: No such extension org.apache.dubbo.rpc.Protocol by name rest, no related exception was found, please check whether related SPI module is missing.


#Tests run (stage 1 in 674 secs.): 3519
#Failed tests (stage 1): 6
#Tests run (stage 2 in 145 secs.): 6
#Failed tests (stage 2): 0
End-to-end time: 819 secs.

